.github/workflows/docs.yml

```diff
@@ -21,7 +21,8 @@ jobs:
       - name: Build docs
         run: nix build -L .#debug.workspaceDocExport
 
-      - name: Deploy docs
+      - if: github.repository == 'fedimint/fedimint'
```

Comment:
Since this job `runs-on: ubuntu-22.04` instead of buildjet, we can allow it to run until the deploy step on forks.